.github/workflows/run-traits-tests.yml

```diff
@@ -44,9 +26,6 @@ jobs:
     - name: Create clean test directory
       run: |
         mkdir testdir
-    - name: Test Traits package (Linux)
-      run: cd testdir && xvfb-run -a python -m unittest discover -v traits
-      if: matrix.os == 'ubuntu-latest'
-    - name: Test Traits package (Windows/macOS)
-      run: cd testdir && python -m unittest discover -v traits
-      if: matrix.os != 'ubuntu-latest'
+    - name: Test Traits package
+      run: python -m unittest discover -v traits
+      working-directory: testdir
```

Comment:
Huh, TIL [`working-directory`](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#defaultsrunworking-directory) context in GH Actions.

Quite handy indeed.